kubetest/gke.go

```diff
@@ -66,6 +66,7 @@ var (
 	gkeDownTimeout                 = flag.Duration("gke-down-timeout", 1*time.Hour, "(gke only) Timeout for gcloud container clusters delete call. Defaults to 1 hour which matches gcloud's default.")
 	gkeRemoveNetwork               = flag.Bool("gke-remove-network", true, "(gke only) At the end of the test remove non-default network that was used by cluster.")
 	gkeDumpConfigMaps              = flag.String("gke-dump-configmaps", "[]", `(gke-only) A JSON description of ConfigMaps to dump as part of gathering cluster logs. Note: --dump or --dump-pre-test-logs flags must also be set. Example: '[{"Name":"my-map", "Namespace":"default", "DataKey":"my-data-key"}]`)
+	gkeDumpLogsCmd                 = flag.String("gke-dump-logs-cmd", "", "(gke-only) if set, run this command to dump cluster logs.")
```

Comment:
Could we rename this flag to sth like `gke-dump-additional-logs` (and hence change the flag description + the log message in `DumpClusterLogs` accordingly)?

In `DumpClusterLogs` function we already collect logs from masters, so such a name would better reflect the flag's semantics.

Reply:
Done.